org.eclipse.dawnsci.remotedataset.server/src/org/eclipse/dawnsci/remotedataset/server/info/TreeServlet.java

```diff
@@ -12,17 +12,20 @@
 package org.eclipse.dawnsci.remotedataset.server.info;
 
 import java.io.IOException;
+import java.security.Provider.Service;
 import java.util.Map;
 
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.xml.ws.ServiceMode;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.xml.ws.ServiceMode'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)